namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -356,7 +357,17 @@ public void run() {
 					if (g == null) {
 						createGroup(NameLayerPlugin.getSpecialAdminGroup(), null, null);
 					} else {
-						removeAllMembers(g.getName());
+						try (Connection connection = db.getConnection();PreparedStatement removeAllMembers = connection.prepareStatement("delete fm.* from faction_member fm "
+								+ "inner join faction_id fi on fi.group_id = fm.group_id where fi.group_name =?;"))  {
+							removeAllMembers.setString(1, g.getName());
+							removeAllMembers.execute();
+						} catch (SQLException e) {
+							e.printStackTrace();
+						}
+						
+						
+						cleanExecute("delete fm.* from faction_member fm inner join groupPlayerTypes gpt on gpt.type_id = fm.type_id "
```

Comment:
so worth noting here; don't modify prior migrations, in general ... I guess in this case it's "ok" but with the new migration system it isn't, as those callables will happen right after the migration that executed, vs. in this case where it happens "later". Also a bit of a regression to remove the use of the clean removeAllMembers call.

What was the intent with the clean execute there? remove all members from the special group from the group types list? query here is incomplete.